node/cn/protocol.go

```diff
@@ -74,7 +77,15 @@ const (
 	ReceiptsRequestMsg = 0x0e
 	ReceiptsMsg        = 0x0f
 
-	MsgCodeEnd = 0x10
+	// Protocol messages belonging to klay/64
+	DummyMsg1 = 0x10
+	DummyMsg2 = 0x11 // it is used for consensus message for istanbul engine
```

Comment:
I can't understand why `DummyMsg2` is added..  it seems duplicated, because `0x11` is already defined at istanbul package. Could you explain what DummyMsg1 & DummyMsg2's role? It would be nice to add a comment about the role of DummyMsgs.

Reply:
So, I wanted to add new messages (staking info messages), but the `0x11` was taken in istanbul engine. That is why I added two dummy messages in `klay` protocol. Please welcome any better suggestion. I will add some comments though.

Reply:
@jeongkyun-oh Oh, I'm just curious that dummyMsgs have some special role.